### PR TITLE
Rename issue #3039

### PR DESCRIPTION
**Pull request by Knapstad**
### Pull Request Checklist

- [x] I’ve read the [guidelines for contributing](https://github.com/getnikola/nikola/blob/master/CONTRIBUTING.rst).
- [x] I updated AUTHORS.txt and CHANGES.txt (if the change is non-trivial) and documentation (if applicable).
- [ ] I tested my changes.

### CODE REVIEW

**Comment by Knapstad** [comment]
Rename DISABLE_INDEXES_PLUGIN_* settings #3039

File: AUTHORS.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-* `Agustin Henze <https://github.com/agustinhenze>`_
+﻿* `Agustin Henze <https://github.com/agustinhenze>`_
```
Comment (felixfontein) [inline AUTHORS.txt]: You inserted a UTF-8 BOM here (a hexdump shows that the file now starts with `ef bb bf`).

Comment (felixfontein) [reply AUTHORS.txt]: (https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8)

Comment (Kwpolska) [reply AUTHORS.txt]: I fixed this for now. In the future, please make sure your editor does not insert BOMs (never use notepad.exe to edit stuff like that)

https://en.wikipedia.org/wiki/Byte_order_mark

File: nikola/nikola.py [hunk 2]
```diff
@@ -714,6 +714,13 @@ def __init__(self, **config):
         if 'UNSLUGIFY_TITLES' in self.config:
             utils.LOGGER.warn('The UNSLUGIFY_TITLES setting was renamed to FILE_METADATA_UNSLUGIFY_TITLES.')
             self.config['FILE_METADATA_UNSLUGIFY_TITLES'] = self.config['UNSLUGIFY_TITLES']
+            
+        if 'DISABLE_INDEXES_PLUGIN_INDEX_AND_ATOM_FEED' in self.config:
+            utils.LOGGER.warn('The DISABLE_INDEXES_PLUGIN_INDEX_AND_ATOM_FEED setting was renamed DISABLE_INDEXES')
+            self.config['DISABLE_INDEXES'] = self.config['DISABLE_INDEXES_PLUGIN_INDEX_AND_ATOM_FEED']
+
+        if 'DISABLE_INDEXES_PLUGIN_RSS_FEED' in self.config:
+            utils.LOGGER.warn('The DISABLE_INDEXES_PLUGIN_RSS_FEED setting was renamed DISABLE_MAIN_RSS_FEED')
```
Comment (Kwpolska) [inline nikola/nikola.py]: You forgot one line here, but I’m going to add it myself.

**Comment by Kwpolska** [comment]
Thanks for contributing, @Knapstad!

**Comment by Knapstad** [comment]
@Kwpolska thank you for letting me :) and thanks for your patiens, its my first contribution out side my own projects

**Pull request merged**